go-controller/pkg/types/const.go

```diff
@@ -200,6 +200,8 @@ const (
 
 	// key for network name external-id
 	NetworkExternalID = OvnK8sPrefix + "/" + "network"
+	// key for network ID which is an integer
+	NetworkIDExternalID = OvnK8sPrefix + "/" + "network-id"
```

Comment:
why are we adding networkID here ? hmm can we just go with what's done for other switches/routers to be consistent and I assume we will need another change to update all switches/routers with network-id right in future right?

Reply:
please feel free to ping me when networkIDs part has been removed..